server/messages/controller.js

```diff
@@ -1,20 +1,51 @@
 var db = require('../db/db.js');
 var User = global.db.User;
 var Item = global.db.Item;
-var Message = global.db.Message;
+var Messages = global.db.Message;
```

Comment:
Why was this renamed to Messages? I used "Message" to be consistent with the Sequelize convention of singular model names.